src/App/PropertyLinks.cpp

```diff
@@ -4667,6 +4886,25 @@ void PropertyXLinkSubList::getLinks(std::vector<App::DocumentObject *> &objs,
     }
 }
 
+void PropertyXLinkSubList::getLinksTo(std::vector<App::ObjectIdentifier>& identifiers,
+                                      App::DocumentObject* obj,
+                                      const char* subname,
+                                      bool all) const
+{
+    if (all || _pcScope != LinkScope::Hidden) {

```

Comment:
This could use early-exit. Generally I'd suggest that the intended (happy) execution path should be the least indented one and all corner cases should be placed on nested blocks.